rpcs3/main.cpp

```diff
@@ -360,6 +362,14 @@ void log_q_debug(QtMsgType type, const QMessageLogContext& context, const QStrin
 	}
 }
 
+template <>
+void fmt_class_string<std::chrono::sys_time<typename std::chrono::system_clock::duration>>::format(std::string& out, u64 arg)
+{
+	std::ostringstream ss;
+	ss << get_object(arg);
```

Comment:
this operator is not supported on gcc yet.

In the meantime you can use something like:
```
    std::time_t dateTime = std::chrono::system_clock::to_time_t(get_object(arg));
    std::tm tm = *std::localtime(&dateTime);
    std::ostringstream ss;
	ss << std::put_time(&tm, "%Y-%m-%e");
    cout<<ss.str();
```

Reply:
with proper format obviously

Reply:
Hm idk, std::localtime is not thread-safe. We either shall wait utill all compilers implement the desired formatting routine or opt-out to a custom implementation.

Reply:
I no longer feel patient enough to await the compilers to implement it.